be/src/exec/broker_scanner.cpp

```diff
@@ -103,6 +103,10 @@ Status BrokerScanner::get_next(Tuple* tuple, MemPool* tuple_pool, bool* eof, boo
         const uint8_t* ptr = nullptr;
         size_t size = 0;
         RETURN_IF_ERROR(_cur_line_reader->read_line(&ptr, &size, &_cur_line_reader_eof));
+        if (_skip_rows > 0){
+            _skip_rows--;
+            continue;
+        }
         if (_skip_next_line) {
```

Comment:
could we replace _skip_next_line by '_skip_rows=1' ?

Reply:
ok, the latest code has been replaced